### DISCUSSION TITLE
What about books that can't be borrowed?

### DISCUSSION THREAD

**Discussion by BuddhaDiedLaughing** [elementdavv/internet_archive_downloader#48] [Q&A]
For some reason - probably bogus challenges - OLD books that should be public domain are not available to borrow.  You can, however, view the full document in the browser.  Pretty irritating.  Below is an example but I've come across a bunch, recently.  I get the sense there are bots abusing the Archive's take-down process.  Probably the biggest retailer of hard copies of books like this - which aren't available through the publisher in print nor digitally - is Amazon.  If you read the papers you know they're not above something like this.  Either way, the public has a right.

Example: https://archive.org/details/chemicalformular0007unse/page/n2/mode/1up

Anyway, thank you for your work.  I wish the browser extension economy were more rewarding to developers.  If you manage to crack this chestnut I'll buy you a coffee.  If you're trying to make actual money maybe you could develop a clever extension for Android?

Cheers

**Comment by robatino** [comment]
I can only see about the first 10 pages and the back cover, at page 11 I get the standard "Limited preview/Some pages are omitted." message.

**Comment by Arcanant** [comment]
This is a good example that the plugin should work but doesn't.

https://archive.org/details/socialpsychology0000katz/mode/2up

**Comment by elementdavv** [comment]
Duplicate of #49.
